src/features/dotnetTest.ts

```diff
@@ -333,7 +333,8 @@ export default class TestManager extends AbstractProvider {
             debuggerEventsPipeName: debuggerEventsPipeName,
             program: program,
             args: args,
-            cwd: cwd
+            cwd: cwd,
+            env: environmentVariables,
```

Comment:
@gregg-miskelly  please review, I am not sure if I need to do additional merging on the existing `result` which is a collection of debugger config, over which the new configuration is spread. Can it have env on it already?

Reply:
No, you shouldn't need to. It is not valid according to the schema ([set here](https://github.com/OmniSharp/omnisharp-vscode/blob/1d71dab8cbd0dfb653fbdb8567dcbf4e3be102d8/package.json#L595)).